packages/components/app/styles/components/pagination-bar.scss

```diff
@@ -2,4 +2,142 @@
 // PAGINATIONBAR
 //
 
-// .hds-pagination-bar {}
+@use "../mixins/focus-ring" as *;
+
+// * Pagination Parent (wrapper) component
+// ----------------------------------------
+.hds-pagination-bar {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 36px;
+  color: var(--token-color-foreground-primary);
+  // font-size: var(--token-typography-body-300-font-size); // = 1rem, not same as 13px, TODO: investigate
+  font-size: 13px;
+}
+
+// * Sub-component: Pagination
+// ----------------------------------------
+.hds-pagination {
+  display: flex;
+  align-self: stretch;
+}
+
+.hds-pagination__page-list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+
+  > li {
+    display: flex;
+  }
+}
+
+.hds-pagination__page-link,
+.hds-pagination__ellipsis {
+  display: flex;
+  align-items: center;
+  padding: 0 12px;
+}
+
+.hds-pagination__page-link {
+  @include hds-focus-ring-with-pseudo-element(
+    $top: 5px,
+    $right: 5px,
+    $bottom: 5px,
+    $left: 5px,
+  );
+  color: inherit;
+  text-decoration: none;
+
+  &:hover {
+    color: var(--token-color-foreground-action-hover);
+  }
+}
+
+.hds-pagination__ellipsis {
+  color: var(--token-color-foreground-faint);
+}
+
+// Hide "Previous" & "Next" labels by default (TODO: Remove from DOM using JS?)
+.hds-pagination__page-label { display: none; }
+
+// NOTE: Trying to account for white space around icons inside the links (will adjust as requested)
+.hds-pagination__page--previous .hds-pagination__page-link { padding-left: 8px; }
+.hds-pagination__page-next .hds-pagination__page-link { padding-right: 8px; }
+
+// "Compact" Pagination Very Rough WIP - TODO: Refactor
+.hds-pagination-bar--is-compact {
+  .hds-pagination__page-label { display: block; }
+
+  .hds-pagination__page-icon + .hds-pagination__page-label,
+  .hds-pagination__page-label + .hds-pagination__page-icon {
+    margin-left: 6px;
+  }
+
+  .hds-pagination__page--previous,
+  .hds-pagination__page--next {
+    display: flex;
+  }
+
+  // may be replaced by JS solution...
+  .hds-pagination__page-list {
+    display: none;
+  }
+}
+
+.hds-pagination__page-current {
+  position: relative;
+  color: var(--token-color-foreground-action);
+
+  &:hover {
+    color: var(--token-color-foreground-action-hover);
+  }
+
+  &:active {
+    color: var(--token-color-foreground-action-active);
+  }
```

Comment:
I'm okay if this is sorted out later on, but for some reason, the color doesn't seem to change when the button is clicked/active

Reply:
Sorry about that, I think I may have messed it up when I refactored how the current page class was applied earlier. It should be fixed now